src/main/java/org/overengineer/inlineproblems/bundles/SettingsBundle.java

```diff
@@ -0,0 +1,35 @@
+package org.overengineer.inlineproblems.bundles;
+
+import com.intellij.DynamicBundle;
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NonNls;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.PropertyKey;
+
+import java.util.function.Supplier;
+
+/**
+ * The SettingsBundle class is a dynamic bundle for managing internationalized messages for the AiCodeBot project.
+ * It extends the DynamicBundle class.
+ *
+ * @author zhengwei
```

Comment:
Who is this? Or where does this code come from?

Reply:
@kuweiguge I cannot merge it without knowing that

Reply:
**kuwei** is my online nickname, **zhengwei** is the pinyin of my Chinese name, and the code is written by me with reference to the official idea plugin example.

Reply:
Refactored author and docstrings in SettingsBundle class

Reply:
Perfect thanks a lot, will be merged when i have some time.